src/components/card.jsx

```diff
@@ -1,14 +1,35 @@
 import React, { Component } from 'react';
 import './style.css';
+import axios from 'axios';
 import ReadMoreReact from 'read-more-react';
 import StarRatingComponent from 'react-star-rating-component';
 
 class cards extends Component {
+  state={
+    clicked:false,
+    popularity:this.props.popularity
+  }
+  handleClick = () => {
+    axios
+      .patch(`/users/update/${this.props.data._id}`, {
+        popularity: this.state.popularity+1
+      })
+      .then(response => {
+        console.log(response)
+      })
+      .catch(function (error) {
+        console.log(error);
+      });
```

Comment:
please handle `res` and `err` properly. And remove the console logs

Phal @ [this](https://itnext.io/javascript-error-handling-from-express-js-to-react-810deb5e5e28)